xml/templates/civicrm_state_province.tpl

```diff
@@ -3994,3 +3994,6 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (NULL, 1081, "SJ", "Samtskhe-Javakheti"),
 (NULL, 1081, "SK", "Shida Kartli"),
 (NULL, 1074, "C", "Central");
```

Comment:
There should be a comma instead of a semi-colon here and closing semi-colon at line no 3999.